docs/background/glossary.md

```diff
@@ -72,9 +72,9 @@ Please note that this Glossary is **work in progress**. Do you encounter missing
 
 - homozygous: an individual in which both alleles at a given locus are **identical** ([MESH](http://www.ncbi.nlm.nih.gov/mesh?term=homozygous)).
 
-- hypermorphic variant: a variant causing the gene product to possess an increased level of activity, or in which the normal (wild-type) gene product is expressed at an increased level
+- hypermorphic variant: a variant characterized by a partial gain of gene activity (including an increase in protein production or function)
```

Comment:
How about:

- hypermorphic variant: a variant characterized by an increase of activity of the gene product, perhaps through increased expression, increased enzymatic activity, or increased stability.

And similarly for hypomorphic. 

I specifically avoided "protein" here because an increase in enzymatic of ribozymes, for example, should still qualify.

Reply:
I agree with @reece about the broader notion of preserving only vocabulary that is used within the specification.

As for the specific definitions, I agree with the changes that @jtdendunnen have made as well as @reece's comment that we should avoid use of the word "protein". I would prefer the use of "gene product" instead. Reece also dropped the use of "partial gain" in favor of "increase" for _hypermorphic_; another change I agree with. There's no notion of a "complete gain" so the use of "partial" here is awkward. "partial loss" for _hypomorphic_ makes sense, since a "complete loss" would be _amorphic_.

Structurally, I prefer @jtdendunnen's use of parentheses in lieu of a clause beginning with "perhaps".

So, aligning these preferences with what is already written, I think I would define these as:

> hypermorphic variant: a variant characterized by a gain of gene activity (e.g., by increased gene product expression or function).

> hypomorphic variant: a variant characterized by a partial loss of gene activity (e.g., by decreased gene product expression or function).

Reply:
I'm not sure why, but my changes didn't seem to stick? I may have messed this up. In summary, like what @jtdendunnen and @reece did here, and would combine them to create the following:

_hypermorphic variant_: a variant characterized by a gain of gene activity (e.g., an increase in gene product expression or function).

_hypormorphic variant_: a variant characterized by a partial loss of gene activity (e.g., a decrease in gene product expression or function).